addons/website_slides/static/src/js/activity.js

```diff
@@ -1,45 +1,47 @@
 odoo.define('website_slides.Activity', function (require) {
-"use strict";
+    "use strict";
 
-var field_registry = require('web.field_registry');
+    const components = {
+        Activity: require('mail/static/src/components/activity/activity.js'),
+    };
 
-require('mail.Activity');
+    const { patch } = require('web.utils');
 
-var KanbanActivity = field_registry.get('kanban_activity');
+    patch(components.Activity, 'website_slides/static/src/components/activity/activity.js', {
```

Comment:
Missing override of `useStore` to introduce extra values used in template. We might actually want a new component for this. See for example in enterprise approval.js we do something very similar with activities.